skopt/acquisition.py

```diff
@@ -4,7 +4,19 @@
 from scipy.stats import norm
 
 
-def _gaussian_acquisition(X, model, y_opt=None, method="LCB",
+def acquisition_1D(X, model, y_opt=None, acq_func="LCB", xi=0.01,
```

Comment:
`acquisition_1D` is not a good name, as it is wrapping `_gaussian_acquisition` only.